src/components/autocomplete/html/autocomplete.tmpl.html

```diff
@@ -0,0 +1,12 @@
+<label>
```

Comment:
What is this template used for? It's missing some of the accessibility stuff from the template in `autocompleteDirective.js`.

Also: I'm still not quite sure why this is wrapped in a label element, since there is no text in it to act as a label.